ports/mimxrt10xx/common-hal/busio/SPI.c

```diff
@@ -234,6 +237,10 @@ void common_hal_busio_spi_deinit(busio_spi_obj_t *self) {
 bool common_hal_busio_spi_configure(busio_spi_obj_t *self,
     uint32_t baudrate, uint8_t polarity, uint8_t phase, uint8_t bits) {
 
+    if (baudrate > 30000000) {
+        baudrate = 30000000; // "Absolute maximum frequency of operation (fop) is 30 MHz" -- IMXRT1010CEC.pdf
```

Comment:
Is this true for the 1060 as well? I bet it is but it'd be good to check.

Reply:
Yes.  I hadn't checked before but I did just now -- the same 30MHz limit is specified in https://www.nxp.com/docs/en/nxp/data-sheets/IMXRT1060IEC.pdf